lib/these.js

```diff
@@ -83,23 +88,48 @@ These.prototype.map = function (f) {
     return new These(this._this, f(this._that));
 };
 
+This.prototype[FL.map] = This.prototype.map;
+That.prototype[FL.map] = That.prototype.map;
+These.prototype[FL.map] = These.prototype.map;
+
 // Apply : ap
 
 This.prototype.ap = function (x) { return x.map(this._this); };
 That.prototype.ap = function (x) { return x.map(this._that); };
 These.prototype.ap = function (x) { return x.map(this._that); };
```

Comment:
Ooops - I just realised `fantasy-check` does not cover `Apply`. Well thanks for pointing this out @davidchambers ! I will fix it.